docs/analyzers.md

```diff
@@ -0,0 +1,34 @@
+# source{d} lookout Analyzers
```

Comment:
look like the name on gitbook is different. And I think it reflects the content of this file better.